src/include/stats.h

```diff
@@ -53,7 +53,47 @@
 
 using json = nlohmann::json;
```

Comment:
Hah!  I had previously been logging the timing and memory data to json but found dealing with that in pandas to be too annoying for my level of pandas expertise, so I went back to fixed-width columns and excised everything about json from my branch.  Will have to add it back but should not be a problem.  Perhaps I can learn enough pandas to use json for my own loging.  It would make some things much much simpler in terms of saving and dumping things and, provided I or someone has sufficient pandas expertise, to process.